simcase-game/src/components/Statement.js

```diff
@@ -0,0 +1,32 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+
+const Container = {
+  textAlign: "center",
+  backgroundColor: "rgb(213,245,245)",
```

Comment:
Will always wonder about combining the various ways to set colors, RGB, HEX, HSL.
Also we need to appropriately use comments to describe functions. Or even JSDoc or something to document.